app/controllers/transcriptions_controller.rb

```diff
@@ -83,7 +83,8 @@ def export_group
     workflow = Workflow.find(params[:workflow_id])
     authorize workflow
 
-    @transcriptions = Transcription.where(group_id: params[:group_id], workflow_id: params[:workflow_id])
+    # Load attached export files as an include with a single extra query
+    @transcriptions = Transcription.with_attached_export_files.where(group_id: params[:group_id], workflow_id: params[:workflow_id])
```

Comment:
is `with_attached_export_files` a method from `ActiveStorage`?

Reply:
Yeah, it's a generated method that preloads the specified relation. The Transcription model has `has_many_attached :export_files`, which sets the model up to use Active Storage on the relation `export_files`, hence `.with_attached_export_files`. 

I found it in the docs here: https://edgeapi.rubyonrails.org/classes/ActiveStorage/Attachment.html

It's the easiest/quickest way I could find to eager load that whole relation in one fell swoop instead of per-transcription.

Reply:
Cool!